amass/sources/twitter_test.go

```diff
@@ -0,0 +1,26 @@
+package sources
+
+import (
+	"testing"
+	
+	"github.com/OWASP/Amass/amass/core"
+)
+
+func TestTwitter(t *testing.T) {
+	if *networkTest == false {
+		return
+	}
+
+	config := setupConfig(domainTest)
+
```

Comment:
Add a check for the twitter API key here.

Reply:
Agreed. I just added the check.